cellbrowser_tools/bin/build_release.py

```diff
@@ -39,29 +48,52 @@ def get_data_groups(prefs):
     # log.info('ABOUT TO CREATE ' + str(total_jobs) + ' JOBS')
     groups = []
     for index, (fovid, group) in enumerate(data_grouped):
-        groups.append(group)
+        groups.append(group.to_dict(orient="records"))
+    log.info("Converted groups to lists of dicts")
+
     return groups
 
 
-@task
 def process_fov_row(group, args, prefs):
-    rows = group.to_dict(orient="records")
-    createJobsFromCSV.do_image(args, prefs, rows)
+    rows = group  # .to_dict(orient="records")
+    log.info("STARTING FOV")
+    try:
+        createJobsFromCSV.do_image(args, prefs, rows)
+    except Exception as e:
+        log.error("=============================================")
+        if args.debug:
+            log.error("\n\n" + traceback.format_exc())
+            log.error("=============================================")
+        log.error("\n\n" + str(e) + "\n")
+        log.error("=============================================")
+        raise
+    log.info("COMPLETED FOV")
 
 
 @task
+def process_fov_rows(groups, args, prefs, distributed_executor_address):
+    # Batch process the FOVs
+    with DistributedHandler(distributed_executor_address) as handler:
+        handler.batched_map(
+            process_fov_row,
+            [g for g in groups],
+            [args for g in groups],
+            [prefs for g in groups],
+            batch_size=30,
```

Comment:
This batch size is pretty small. Let `DistributedHandler` use default? Especially because you have 100 workers.